telemetry/config.go

```diff
@@ -0,0 +1,48 @@
+package telemetry
+
+import "time"
+
+type OtelConfig struct {
+	Disable                 bool          `mapstructure:"disable"`
+	CollectorEndpoint       string        `mapstructure:"collector-endpoint"`
+	CollectorMetricsURLPath string        `mapstructure:"collector-metrics-url-path"`
+	User                    string        `mapstructure:"user"`
+	Token                   string        `mapstructure:"token"`
+	PushInterval            time.Duration `mapstructure:"push-interval"`
+}
+
+var DefaultOtelConfig = OtelConfig{
```

Comment:
Since most nodes will not be "updating" their configs using the template as they are just existing nodes that are updating their binary, will these default config values still work?

Reply:
yep, this config is injected in `root.go` and is only updated if they set the values themselves in the toml